service/k8sclient/k8sclient.go

```diff
@@ -319,29 +319,21 @@ type K8sClient struct {
 }
 
 func init() {
+	pmmClientImage = "perconalab/pmm-client:dev-latest"
+
 	pmmClientImageEnv, ok := os.LookupEnv("PERCONA_TEST_DBAAS_PMM_CLIENT")
 	if ok {
 		pmmClientImage = pmmClientImageEnv
 		return
 	}
 
-	if pmmversion.PMMVersion == "" {
-		// Prevent panicing on local development builds.
-		pmmClientImage = "perconalab/pmm-client:dev-latest"
-		return
-	}
-
-	v, err := goversion.NewVersion(pmmversion.PMMVersion)
-	if err != nil {
-		panic("failed to decide what version of pmm-client to use: " + err.Error())
-	}
+	if pmmversion.PMMVersion != "" {
+		v, err := goversion.NewVersion(pmmversion.PMMVersion)
+		if err != nil {
+			panic("failed to decide what version of pmm-client to use: " + err.Error())
```

Comment:
let's not panic, I think it's better to log an error and use dev-latest

Reply:
:+1: